nucleus/metrics/cuboid_metrics.py

```diff
@@ -10,6 +11,9 @@
 from .filtering import ListOfAndFilters, ListOfOrAndFilters
 from .filters import confidence_filter
 
+DEFAULT_IOU_THRESHOLD = 0.1
```

Comment:
Why are we changing the default from `0.0` to `0.1`? I'm not against it would just be nice to have a comment explaining the choice of the default.

Reply:
the default of 0.0 is really confusing in my opinion, in particular because it makes predictions with zero overlap true positives.  0.1 seemed reasonable for 3D IoU, though I wouldn't be opposed to 0.5 (to match the 2D default threshold) - what do you think?